project-fortis-pipeline/ops/install-fortis-services.sh

```diff
@@ -123,7 +131,79 @@ status:
 EOF
 kubectl create -f "${deployment_yaml}","${service_yaml}"
 
-# request a public ip for the service
-kubectl expose deployment project-fortis-services \
-  --type "LoadBalancer" \
-  --name "project-fortis-services-lb"
+if [ "${endpoint_protection}" = "none" ]; then
```

Comment:
Elsewhere uses double equals for string equality. While they are equivalent ([source](http://tldp.org/LDP/abs/html/comparison-ops.html#SCOMPARISON1)) it would be best to stick to using double-equals everywhere to avoid confusion.